### ISSUE TITLE
KeyError when search for "excludible" with -y

### ISSUE THREAD

**Issue by Pezhin** [Pezhin/terdict#1]
Trace back
```
excludible
Traceback (most recent call last):
  File "/usr/local/bin/terd", line 9, in <module>
    load_entry_point('terdict==0.1', 'console_scripts', 'terd')()
  File "build/bdist.macosx-10.11-x86_64/egg/terdict/terdict.py", line 76, in main
  File "build/bdist.macosx-10.11-x86_64/egg/terdict/terdict.py", line 31, in process
  File "build/bdist.macosx-10.11-x86_64/egg/terdict/Translator/YoudaoTranslator.py", line 28, in processWord
  File "build/bdist.macosx-10.11-x86_64/egg/terdict/Translator/YoudaoTranslator.py", line 36, in searchWithEnglish
KeyError: 'basic'
```

**Comment by Pezhin** [comment]
Bug cause: not handling the response array well.

**Comment by Pezhin** [comment]
fixed by #1cc0897
